test/Ocelot.UnitTests/DownstreamRouteFinder/DownstreamRouteFinderTests.cs

```diff
@@ -27,6 +29,8 @@ public class DownstreamRouteFinderTests
         private string _upstreamHttpMethod;
         private string _upstreamHost;
         private string _upstreamQuery;
+        private UpstreamHeaderRoutingOptions _upstreamHeaderRoutingOptions;
+        private IHeaderDictionary _requestHeaders;

```

Comment:
```suggestion
        private readonly IHeaderDictionary _requestHeaders;
```
But a small refactoring of private helpers are required.